src/typings.ts

```diff
@@ -356,14 +360,33 @@ export interface DecimalsObject {
 export interface UserObject {
   address: string
   pubKey: string
-  keystore: string
+  backup: string
+}
+
+export interface DeployedIdentity {
+  identity: string
+  nextNonce: number
+  balance: string
 }
 
 export interface Signature {
   ecSignature: ECSignature
   concatSig: string
 }
 
+export interface IdentityBackup {
```

Comment:
It is not a backup, it is just a format that can be used to store the wallet for any reason.